app/components/alert-prompt/component.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import classnames from 'classnames';
+
+import Modal, { ModalProps } from 'components/modal';
+import Button from 'components/button';
+import Icon, { IconProps } from 'components/icon';
+
+export interface AlertPromptProps {
```

Comment:
Would you mind moving this interface definition to a `types.ts` file in the local folder?